python/res/job_queue/job_queue_node.py

```diff
@@ -27,14 +27,15 @@ class JobQueueNode(BaseCClass):
     _get_status = ResPrototype("int job_queue_node_get_status(job_queue_node)")
     _update_status = ResPrototype("bool job_queue_node_update_status_simple(job_queue_node, driver)")
     _set_status = ResPrototype("void job_queue_node_set_status(job_queue_node, int)")
+    _get_submit_attempt = ResPrototype("int job_queue_node_get_submit_attempt(job_queue_node)")
 
     def __init__(self,job_script, job_name, run_path, num_cpu, 
                     status_file, ok_file, exit_file, 
                     done_callback_function, exit_callback_function, callback_arguments):
         self.done_callback_function = done_callback_function
         self.exit_callback_function = exit_callback_function
         self.callback_arguments = callback_arguments
-    
+        self.stopped_by_user = False
```

Comment:
Should we perhaps rename this for `stopped`? And in addition, maybe this should be a private member `_stopped`.

Reply:
I think it's just a legacy from the past. Agree, this could be renamed.

Reply:
fixed now